mev_inspect/arbitrages.py

```diff
@@ -105,6 +115,9 @@ def _get_shortest_route(
         return None
 
     shortest_remaining_route = None
+    max_remaining_route_length = (
```

Comment:
```max_remaining_route_length``` implies that the variable represents the longest possible route that is still viable - which makes sense - besides the fact that initial assignment is ```None``` - which is meaningless. What about ```current_shortest_route_length```? Then, ```None``` indicates there is no current shortest route, until the function is called at runtime

Reply:
`current_shortest_route_length` isn't accurate because it isn't the shortest route. It's the shortest of the rest of the route (shortest continuation) - so it'd be:
`current_shortest_remaining_route_length`

but still max and shortest are slightly distinct

i moved away from shortest because it ends up getting messy looking in the logic

ex: if the current shortest path is 4, we should only be looking for paths that are max length 3

so you have to change all the comparisons to be "one less than the shortest length"

v. max says "Optionally specify a max length for a shortest path"

Reply:
what about ```next_shortest_route_length```?

Reply:
the remaining is also a max though since on the recursion it becomes `max_route_length`
the len of